### PR TITLE
Feature/accordion and map interactions

### PR DESCRIPTION
**Pull request by cazapatamar**
New component called Landscape

### CODE REVIEW

File: src/search/CompensationFactor.jsx [hunk 1]
```diff
@@ -14,6 +14,10 @@ class CompensationFactor extends React.Component {
     };
   }
 
+  /**
+   * Show or hide the detailed information on each graph
+   *
```
Comment (erikasv) [inline src/search/CompensationFactor.jsx:19]: delete this line

File: src/humanFootprint/TimelineFootprint.jsx [hunk 2]
```diff
@@ -70,6 +70,10 @@ class TimelineFootprint extends React.Component {
     };
   }
 
+  /**
+   * Show or hide the detailed information on each graph
+   *
```
Comment (erikasv) [inline src/humanFootprint/TimelineFootprint.jsx:75]: delete this line

File: src/humanFootprint/PersistenceFootprint.jsx [hunk 3]
```diff
@@ -19,6 +19,10 @@ class PersistenceFootprint extends React.Component {
     };
   }
 
+  /**
+   * Show or hide the detailed information on each graph
+   *
```
Comment (erikasv) [inline src/humanFootprint/PersistenceFootprint.jsx:24]: delete this line

File: src/landscape/Landscape.jsx [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+   *
+   */
+  handlerAccordionGeometry = (level, expandedTab) => {
+    const { handlerSwitchLayer } = this.props;
+    const { expandedLevel1, expandedLevel2 } = this.state;
+
+    switch (level) {
+      case '1':
+        this.setState({ expandedLevel1: expandedTab });
+        if (expandedTab === 'hfp') {
```
Comment (alegnaaived) [inline src/landscape/Landscape.jsx:36]: The idea was to replace hfp to hf, isn't?

**Pull request merged**